assets/agenda/components/AgendaListItemIcons.jsx

```diff
@@ -57,10 +57,28 @@ class AgendaListItemIcons extends React.Component {
         };
     }
 
+    getItemByschema(props) {
+        const {listConfig} = props;
+        const {scheme} = listConfig && listConfig.subject || {};
+
+        if (scheme) {
+            const subjects = getSubjects(props.item);
+            const filteredSubjects = subjects.filter(item => {
+                return Array.isArray(scheme) ? scheme.includes(item.scheme) : item.scheme === scheme;
+            });
+            return filteredSubjects;
+        }
+        if (listConfig.subject) {
```

Comment:
Here you're assuming `listConfig` is an object, but in line 62 you're covering the scenario if `listConfig` is not defined, in which case this line would throw an exception.

It might be better to surround this entire `getSubjects` section in an outer `if (listConfig && listConfig.subject)`, such as:
```javascript
if ((listConfig || {}).subject != null) {
    const scheme = listConfig.subject.scheme;
    const subjects = getSubjects(props.item);
    return scheme == null ?
        subjects :
        subjects.filter((item) => Array.isArray(scheme) ?
            scheme.includes(item.scheme) :
            item.scheme === scheme
        );
}

return [];
```